The wildcard specification isn't permitted by newer versions of pip/setuptools.

--- a/setup.py	Fri Nov  6 11:04:27 2020
+++ b/setup.py	Mon Oct  2 15:53:09 2023
@@ -31,7 +31,7 @@
 install_requires = [
     'pycurl',
     'pyparsing<2.4.2;python_version<="3.4"',
-    'pyparsing>=2.4*;python_version>="3.5"',
+    'pyparsing>=2.4.2;python_version>="3.5"',
     'six',
     'configparser;python_version<"3.5"',
     'chardet',
